src/index.ts

```diff
@@ -333,6 +333,18 @@ function main() {
                 })
                 .then((ans: Answers) => {
                     if (program.sku.toLowerCase() === solutionSkus[solutionSkus.local]) {
+                        const dateTime = Date.now().toString();
+                        answers.azureWebsiteName = 'local-deployment-' + dateTime;
```

Comment:
we can still use solutionName as website prefix here.

Reply:
yes, perfect